readthedocs/proxito/views/hosting.py

```diff
@@ -504,6 +534,9 @@ def _v1(self, project, version, build, filename, url, user):
             )
 
         # Update this data with the one generated at build time by the doctool
+        # TODO: this looks a little dangerous.
+        # We can't trust this output, we may want
+        # to scope the data that can be overridden.
         if version and version.build_data:
             data.update(version.build_data)
```

Comment:
We can remove this code for now. It was part of the design and we were testing this pattern, but I don't think we will use it in the end. We can come back to it if required, tho.